sample-apps/spark/src/main/java/com/amazon/sampleapp/App.java

```diff
@@ -55,7 +55,9 @@ public static void main(String[] args) {
     get(
         "/outgoing-http-call",
         (req, res) -> {
-          logger.info("Executing outgoing-http-call");
+          if (System.getenv("SAMPLE_APP_LOG_LEVEL") == "INFO") {
```

Comment:
Create a `private static final boolean sampleAppLog = "info".equals(System.getenv("SAMPLE_APP_LOG_LEVEL"))` to avoid evaluating this every time.

And think lowercase is better for the value
And we need to use `.equals`

Reply:
Spoke offline that `INFO` upper case works well too. Thanks for the help! Changes made!